.travis.yml

```diff
@@ -1,28 +1,16 @@
 language: r
 r: 
-  - 3.4.2
+  - 4.1.1
```

Comment:
Is this also a reference to R version and does it also need to be 4.1.2?

Reply:
will update for now but please note the travis CI should be updated to github actions (raised a ticket for this in our Teams area)